packages/terra-badge/src/terra-dev-site/doc/example/BadgeVisuallyHiddenText.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import Badge from 'terra-badge/lib/Badge';
+
+const BadgeVisuallyHiddenText = () => (
+  <div>
+    <Badge text="Low" visuallyHiddenText="Rick Factor" />
```

Comment:
Nit: `visuallyHiddenText` should probably be `Risk Factor`

Reply:
We could just call this a Rick & Morty badge :)